packages/lesmis-server/src/utils/verifyLoginAccess.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { AccessPolicy } from '@tupaia/access-policy';
+import { PermissionsError } from '@tupaia/utils';
+import { LESMIS_PERMISSION_GROUP } from '../constants';
+
+export const verifyLoginAccess = (accessPolicy: AccessPolicy) => {
+  const hasAccess = new AccessPolicy(accessPolicy.policy).allowsAnywhere(LESMIS_PERMISSION_GROUP);
```

Comment:
Is it this, or is it "has at least Public access to Laos"? I guess this is better in terms of actually being able to log in. Yep, I'm on board.